api/src/main/java/org/openmrs/api/handler/ExistingOrNewVisitAssignmentHandler.java

```diff
@@ -112,14 +112,15 @@ private static VisitType loadVisitType(EncounterType encounterType) throws APIEx
 			
 			VisitService visitService = Context.getVisitService();
 			String targetEncounterTypeId = encounterType.getId().toString();
+			String targetEncounterTypeUuid = encounterType.getUuid();
 			
 			String[] mappings = value.split(",");
 			for (String mapping : mappings) {
 				int index = mapping.indexOf(':');
 				if (index > 0) {
 					String encounterTypeIdOrUuid = mapping.substring(0, index).trim();
-					if (targetEncounterTypeId.equals(encounterTypeIdOrUuid)
-					        || encounterType.getUuid().equals(encounterTypeIdOrUuid)) {
+					if ("default".equals(encounterTypeIdOrUuid) || targetEncounterTypeId.equals(encounterTypeIdOrUuid) 
```

Comment:
Can you describe what you are trying to achieve with `default`?

Reply:
What I understood from the ticket [TRUNK-5007](https://issues.openmrs.org/browse/TRUNK-5007) and ticket [EA-116](https://issues.openmrs.org/browse/EA-116) is that the `default` wildcard should be used if no specific encounter type is matched  and `beforeCreateEncounter()` from `ExistingOrNewVisitAssignmentHandler` should be able to handle the `default` wildcard `EncounterType` from the `globalProperty`mappings value.